resources/shaders/dxr_pathtracer_functions.hlsl

```diff
@@ -181,11 +220,19 @@ float3 ggxIndirect(float3 hit_pos, float3 fN, float3 N, float3 V, float3 albedo,
 #endif
 
 		// #### BOUNCE ####
-		float3 irradiance = TraceColorRay(hit_pos + (EPSILON * N), L, depth, seed);
+		float3 irradiance = 0;
+		if (use_raycone)
+		{
+			irradiance = TraceColorRayCone(hit_pos + (EPSILON * N), L, depth, seed, cone);
+		}
+		else
+		{
+			irradiance = TraceColorRay(hit_pos + (EPSILON * N), L, depth, seed);
+		}
 		if (dot(N, L) <= 0.0f) irradiance = float3(0, 0, 0);
```

Comment:
Perhaps split this up into multiple lines? The statement right after the check could potentially be a place for future bugs due to the lack of "{" and "}".

Reply:
not new code.

Reply:
So it's not worth resolving? If not, feel free to commit the file the way it is. :)

Reply:
not new code.